modules/claims_api/app/controllers/concerns/claims_api/v2/disability_compensation_validation.rb

```diff
@@ -833,8 +834,51 @@ def raise_exception_if_value_not_present(val, form_obj_description)
         )
       end
 
+      def validate_claim_process_type_bdd!
+        claim_process_type = form_attributes['claimProcessType']
+        if claim_process_type == 'BDD_PROGRAM'
+          date = form_attributes['claimDate'] || Time.find_zone!('Central Time (US & Canada)').today
```

Comment:
for curiosity, what is the decision for Central Time here? not sure I see anything in the ticket so just getting clarity.

Reply:
See validate_form_526_submission_claim_date comment, line 105 in the same file:
 # EVSS runs in the Central US Time Zone.
 # So 'claim_date' needs to be <= current day according to the Central US Time Zone.